projects/convex/index.js

```diff
@@ -132,7 +132,7 @@ function sumSingleBalance(
   token,
   balance
 ) {
-    const prevBalance = ethers.BigNumber.from(balances[token] ?? "0");
+    const prevBalance = ethers.BigNumber.from(balances[token] || "0");
```

Comment:
This actually isn't equivalent, the double question mark "nullish coalescing" is a relatively new feature ES2010 which will take the right side if the left is null or undefined. So the truth table doesn't behave the same.
```js
> [0, 1, false, true, null, undefined].map((value) => value ?? '0')
[ 0, 1, false, true, '0', '0' ]
> [0, 1, false, true, null, undefined].map((value) => value || '0')
[ '0', 1, '0', true, '0', '0' ]
```
I don't think it mater particularly in this case, but I wish we had unit test coverage so we could know if one introduce regression over the other

Reply:
For sure, definitely familiar with `??` from TypeScript, node.green shows it's available in Node 16...It seemed like `balances[token]` was reasonably expected to be undefined so indeed changing to `||` *should* have the same net effect here. `npm test` worked with convex.

What do you think about checking w/`ethers.BigNumber.isBigNumber`?

Reply:
Yeah definitely. When I saw the commit title I thought you were saying the opposite, like `||` was newer hence better to use, that's where I got confused :facepalm: 
Checking with isBigNumber is certainly safer indeed.
Things is sometime making it more right can make things wrong, I'd personally be more confident refactoring things around after we increased some of the coverage we (don't) have

Reply:
Oh no! Oops - my bad.